docs/single-command-install.md

```diff
@@ -0,0 +1,64 @@
+# Single-Command Installation Guide
+
+This document provides comprehensive instructions for the quick, single-command installation of the EDA Server Operator. Also covered are additional details such as prerequisites, uninstallation, and troubleshooting tips.
+
+## Prerequisites
+Before proceeding with the installation, ensure that the following prerequisites are met:
+
+1. **Kubernetes Cluster**: You need an active Kubernetes cluster. If you do not have one, you can set it up using platforms like Minikube, Kind, or a cloud provider like AWS, Azure, or GCP.
+
+2. **kubectl**: The Kubernetes command-line tool, kubectl, should be installed and configured to communicate with your cluster. You can check its availability by running kubectl version.
+
+## Installation
+The EDA Server Operator can be installed using a single command. This command applies a YAML file from the EDA Server Operator's GitHub repository directly to your Kubernetes cluster.
+
+Run the following command in your terminal, modifying the version to whichever version you want to install.
+
+```bash
+kubectl apply -f https://github.com/ansible/eda-server-operator/releases/download/1.0.0/operator.yaml
+```
+
+> [!Note]
+> This will create the EDA Server Operator resources in the eda-server-operator-system namespace.
+
+Now create your EDA custom restore by applying the `eda-demo.yml` file and you will soon have a working EDA instance!
```

Comment:
Providing example of `eda-demo.yaml` helps a lot. For example, just duplicate `eda.yaml` in `README.md`.

````suggestion
Now create your EDA custom restore by applying the `eda-demo.yml` file and you will soon have a working EDA instance!

```yaml
# eda-demo.yaml
apiVersion: eda.ansible.com/v1alpha1
kind: EDA
metadata:
  name: my-eda
spec:
  automation_server_url: https://awx-host
```
````